applications/nrf_desktop/README.rst

```diff
@@ -400,6 +400,7 @@ The following build types are available for various boards in the nRF Desktop:
 * ``b0`` -- ``debug`` build type with the support for the B0 bootloader enabled (for :ref:`background DFU <nrf_desktop_bootloader_background_dfu>`).
 * ``mcuboot`` -- ``debug`` build type with the support for the MCUboot bootloader enabled (for :ref:`serial recovery DFU <nrf_desktop_bootloader_serial_dfu>` or :ref:`background DFU <nrf_desktop_bootloader_background_dfu>`).
 * ``shell`` -- ``debug`` build type with the shell enabled.
+* ``wwcb`` -- ``debug`` build type working with ChromeBook (WWCB).
```

Comment:
Since it will have B0 bootloader, maybe we could call it `prj_b0_wwcb`? In general we explicitly inform about used bootloader in the configuration name.